code/core/src/shared/checklist-store/checklistData.tsx

```diff
@@ -1091,12 +1097,14 @@ export const Disabled: Story = {
                 and a description.
               </p>
               <CodeSnippet language="typescript">
```

Comment:
I guess it makes no difference, but the `language` is still set here to `typescript`